### ISSUE TITLE
Wheel 0.37.1 SHA changed when 0.38 was released

### ISSUE THREAD

**Issue by ZacharyThomas** [pypa/wheel#469]
Hi all,

We use pipenv to install dependencies. Wheel is a transitive dependency which is locked to 0.37.1. It seems like on release of 0.38, the SHA changed. Is this intentional?

```
[pipenv.exceptions.InstallError]: Collecting wheel==0.37.1
[pipenv.exceptions.InstallError]:   Using cached wheel-0.37.1-py3-none-any.whl (35 kB)
[pipenv.exceptions.InstallError]: ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
[pipenv.exceptions.InstallError]:     wheel==0.37.1 from https://files.pythonhosted.org/packages/53/fa/dcb07518f988a02de7a5dc5f0706555c203177fc9291c583fa021ffaec9c/wheel-0.37.1-py3-none-any.whl (from -r /tmp/pipenv-la9xfvf9-requirements/pipenv-fb_ilcyy-requirement.txt (line 1)):
[pipenv.exceptions.InstallError]:         Expected sha256 4bdcd7d840138086126cd09254dc6195fb4fc6f01c050a1d7236f2630db1d22a
[pipenv.exceptions.InstallError]:         Expected     or e9a504e793efbca1b8e0e9cb979a249cf4a0a7b5b8c9e8b65a5e39d49529c1c4
[pipenv.exceptions.InstallError]:              Got        bbb2658b04929e445a903697c9588dd7d20b1b5f9e717843b62963ac92be502d
```

**Comment by agronholm** [comment]
I forgot to update the version number if `__init__.py` when tagging the release. I discovered this mistake quickly when the upload failed. I've removed the newly added file from the previous release now. Let me know if this didn't fix the issue.

**Comment by agronholm** [comment]
It's published but yanked because the setuptools dependency caused issues. I'm hoping that we can fix this with the public API effort.

**Comment by jykae** [comment]
@agronholm thanks for fast answer, when 0.38.0 would be available? just pipenv check nagging about this 88f02bc

**Comment by agronholm** [comment]
You can try to install 0.38.0 by specifying the exact version in your dependencies. It won't ever be un-yanked, but if I figure out a quick fix, I'll publish 0.38.1 instead.
